.github/workflows/tests-polkadot-deps.yml

```diff
@@ -58,7 +58,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [18.x, 20.x]
+        node-version: [18.x, 20.x, 22.x]
```

Comment:
22 is not yet an active LTS, I'd say we make it optional for now. For this, add it to the 'include'